### ISSUE TITLE
Visual Studio Solution files build libraries with incorrect names.

### ISSUE THREAD

**Issue by adriennea** [OpenMAMA/OpenMAMA#74]
Replacing [BZ-90] (http://bugs.openmama.org/show_bug.cgi?id=90)

It appears the solution files for Visual Studio build the common, OpenMAMA and OpenMAMA-CPP libraries with incorrect names. These should follow the convention found in SCons:

| Visual Studio Name | SCons Name |
| ---------------------- | --------------------|
| commonc.lib         |  libwombatcommonc.lib/dll |
| mamac.lib           |  libmamac.lib/dll         |
| mamacpp.lib         |  libmamacpp.lib/dll       |

Solution files should be updated to reflect this.

**Comment by MattMulhern** [comment]
code changes being tracked on #132, but leaving this here to track issue separately.

**Comment by MattMulhern** [comment]
Seems fine now after 19af5175604e892e1ab5e15e1ca2062fdc0187f3 (Part of PR #132)
```
D:\mmulhern\openmama\vs_update>dir x64\Debug
 Volume in drive D has no label.
  Volume Serial Number is 94C5-7CD7

   Directory of D:\mmulhern\openmama\vs_update\x64\Debug

   10/02/2016  13:53    <DIR>          .
   10/02/2016  13:53    <DIR>          ..
   10/02/2016  13:50           175,346 libmamac.exp
   10/02/2016  13:50           289,116 libmamac.lib
   10/02/2016  13:50         2,026,496 libmamac.pdb
   10/02/2016  13:50           754,176 libmamacmdd.dll
   10/02/2016  13:50         1,222,892 libmamacmdd.ilk
   10/02/2016  13:51           384,323 libmamacpp.exp
   10/02/2016  13:51           635,980 libmamacpp.lib
   10/02/2016  13:51         4,631,552 libmamacpp.pdb
   10/02/2016  13:51           883,200 libmamacppmdd.dll
   10/02/2016  13:51         8,286,904 libmamacppmdd.ilk
   10/02/2016  13:53            24,706 libwombatcommonc.exp
   10/02/2016  13:53            43,562 libwombatcommonc.lib
   10/02/2016  13:53           535,552 libwombatcommonc.pdb
   10/02/2016  13:53             1,199 libwombatcommoncpp.exp
   10/02/2016  13:53             2,834 libwombatcommoncpp.lib
   10/02/2016  13:53           371,712 libwombatcommoncpp.pdb
   10/02/2016  13:53            37,888 libwombatcommoncppmdd.dll
   10/02/2016  13:53           323,184 libwombatcommoncppmdd.ilk
   10/02/2016  13:53           134,656 libwombatcommonmdd.dll
   10/02/2016  13:53           604,840 libwombatcommonmdd.ilk
   10/02/2016  13:53            63,063 libwombatcommonmdd.map
```

**Comment by MattMulhern** [comment]
Fix for this went in with #134, closing off.
